appengine/standard/endpoints/backend/main_test.py

```diff
@@ -12,42 +12,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import endpoints
 import main
 import mock
+import unittest
+from google.appengine.ext import testbed
 from protorpc import message_types
 
+class EchoTestCase(unittest.TestCase):
```

Comment:
please use pytest style tests. Refer to the test you overwrote here for an example.